### PR TITLE
Create ccp-spain.toml

### PR DESCRIPTION
**Pull request by rbevansp**
Parser for CVKMNLC COVID-19 patient data, Hospital del Mar, Barcelona, Spain. 
Needs review. Thanks!

### CODE REVIEW

**Review by ekamau — COMMENTED** [review]
looks good .. 
just wondered if some of the observations like oxygen_saturation and temperature might have been taken during the study? i.e., phase = "study" rather than "admission" .. is it worth checking?

**Comment by rbevansp** [comment]
@ekamau thanks for picking up on this! 
I've double checked and it appears that all of the 'observations' are indeed recorded at the time of admission. 
During the hospitalisation only what we consider as 'treatments' are recorded (e.g. specific drugs).

File: isaric/parsers/ccp-spain.toml
Comment (sadiekelly) [inline isaric/parsers/ccp-spain.toml:1]: hi @rbevansp here's my review file for the Spain parser - let me know if you want to chat through anything - thanks!
[Spain parser review 2023-08-02.xlsx](https://github.com/globaldothealth/isaric/files/12241988/Spain.parser.review.2023-08-02.xlsx)

Comment (sadiekelly) [reply isaric/parsers/ccp-spain.toml:1]: @rbevansp updates all look good to me - thanks!

**Pull request merged**